src/Entity/EntityId.php

```diff
@@ -179,4 +191,10 @@ public function equals( $target ) {
 			&& $target->serialization === $this->serialization;
 	}
 
+	protected function getRepositoryNameAndLocalPart( $serialization ) {
```

Comment:
This could be static, and perhaps should be.
Also, the name sounds like a getter, which this is not. Perhaps "computeRepositoryNameAndLocalPart" - or "splitIntoRepositoryNameAndLocalPart".
Also, please document contract and return type.

Reply:
True. Renamed to extractRepositoryNameAndLocalPart, and made static.